### PR TITLE
application tier

### PR DESCRIPTION
**Pull request by hjacobs**
Make the `criticality_level` (tier) field nullable.

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

File: resources/api/kio-api.yaml [hunk 1]
```diff
@@ -564,11 +564,10 @@ definitions:
...
         example: https://github.com/zalando-stups/kio/issues
       criticality_level:
-        description: Criticality level of an application
+        description: Criticality level (tier) of an application
         type: integer
-        minimum: 1 # low
-        maximum: 3 # high
-        default: 2 # medium
+        minimum: 1 # Tier-1
+        maximum: 3 # Tier-3
```
Comment (ThorbjoernG) [inline resources/api/kio-api.yaml]: max should be 4 (for not relevant)
```suggestion
        maximum: 4 # not relevant
```

**Comment by ThorbjoernG** [comment]
👍

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**